File: app/src/main/java/com/alyndroid/pokemon/model/Pokemon.java
package com.alyndroid.pokemon.model;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "fav_table")
public class Pokemon {
    @PrimaryKey(autoGenerate = true)
    private int id;
    private String name;
    private String url;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public Pokemon(String name, String url) {
        this.name = name;
        this.url = url;
    }
}

File: app/src/main/java/com/whiteside/bokemoncourse/model/Pokemon.kt
package com.whiteside.bokemoncourse.model

import androidx.annotation.NonNull
import androidx.databinding.BaseObservable
import androidx.databinding.Bindable
import androidx.room.Entity
import androidx.room.PrimaryKey
import com.whiteside.bokemoncourse.BR


@Entity(tableName = "fav_pokemons")
class Pokemon : BaseObservable() {

    @NonNull
    @PrimaryKey
    lateinit var name: String

    @get:Bindable
    var url: String? = null
        set(value) {
            field = value
            notifyPropertyChanged(BR.url)
        }
}
